installer/PowerToysSetup/Product.wxs

```diff
@@ -372,7 +372,6 @@
           <netfx:NativeImage Id="FancyZonesEditor.exe" Platform="64bit" Priority="0" />
         </File>
         <File Source="$(var.BinX64Dir)modules\$(var.FancyZonesProjectName)\ControlzEx.dll" />
-        <File Source="$(var.BinX64Dir)modules\$(var.FancyZonesProjectName)\MahApps.Metro.dll" />
```

Comment:
This is still need as this is for FancyZones

Reply:
Damn, search and delete junkie...